### ISSUE TITLE
Feature request: Add ansible like variable substitution in workflow.yaml

### ISSUE THREAD

**Issue by Panky-codes** [refenv/cijoe#49]
Currently, we can only access the config variables only via worklets using `cijoe.config.options..`.

Having a feature to directly substitute the variable in workflow.yaml itself would be helpful and avoid bolier plate.
Example behaviour:
```
# local.config
fio:
  bin: "/usr/local/bin"
# fio.workflow
- name: Run fio
  run: |
    {{ config.options[fio][bin] }} --io_depth=4 ....
```

**Comment by safl** [comment]
This is a duplicate of: https://github.com/refenv/cijoe/issues/45 closing https://github.com/refenv/cijoe/issues/45 since this current issue has a better description.

**Comment by safl** [comment]
@Panky-codes this was fixed in #45 thus closing it
